Project/Engine/src/sysFrame/System/Json/Json.cpp

```
@@ -0,0 +1,148 @@
+    auto& data = datas_[_path];
+
+    Object object;
+    object[_key] = _value;
+
+    Group group;
+	group[_group] = object;
+
+    data.push_back(group);
+}
+
+Json::Value Json::GetValue(const std::string& _path, const std::string& _group, const std::string& _key) const {
```

Cppcheck (2.10-2)

[style] The function 'GetValue' is never used.

(unusedFunction)